tekton/README.md

```diff
@@ -12,3 +12,15 @@ Resources are organised in folders:
 - The [resources](resources/README.md) folder contains Tekton resources used for
   various automation tasks: building container images, doing releases,
   maintaining the GitHub org and more.
+
+# Secrets
+
+Some of the resources require secrets to operate.
+- `GitHub` secrets: `bot-token-github` used for syncing label configuration and
+  org configuration requires, `github-token` used to create a draft release
+- `GCP` secrets: `nightly-account` is used by nightly releases to push releases
+  to the nightly bucket. It's a token for service account
+  `release-right-meow@tekton-releases.iam.gserviceaccount.com`.
+  `release-secret` is used by Tekton Pipeline to push pipeline artifacts to a
+  GCS bucket. It's also used to push images built by cron trigger (or Mario)
+  to the image registry on GCP.
```

Comment:
btw there was an attempt to document some of this at https://github.com/tektoncd/plumbing/blob/0eb8f8bbfd7db6a8f1fdc4a54221140b126464cd/prow/README.md#creating-the-secrets as well, would be great to maybe remove that and point it here or point them both to a new `secrets.md` or something

Reply:
OK - I thought I'd keep prow's cluster secrets in prow/README.md and dogfooding's cluster secrets in tekton/README.md, but I could as well merge them into one secrets.md if you prefer so.

Reply:
I think it's fine to keep them separate for now :angel: